dotCMS/src/main/java/com/dotmarketing/portlets/workflows/model/WorkflowAction.java

```diff
@@ -1,155 +1,273 @@
 package com.dotmarketing.portlets.workflows.model;
 
-import java.io.Serializable;
-import java.util.ArrayList;
-import java.util.List;
-
-
 import com.dotcms.repackage.com.fasterxml.jackson.annotation.JsonIgnore;
 import com.dotmarketing.business.PermissionAPI;
 import com.dotmarketing.business.PermissionSummary;
 import com.dotmarketing.business.Permissionable;
 import com.dotmarketing.business.RelatedPermissionableGroup;
 import com.dotmarketing.exception.DotDataException;
 import com.dotmarketing.util.UtilMethods;
+import java.io.Serializable;
+import java.util.*;
 
-
+/**
+ *
+ *
+ * @author root
+ * @version 1.x
+ * @since Mar 22, 2012
+ */
 public class WorkflowAction implements Permissionable, Serializable{
+
+	// requires lock options:
+	public static final String LOCKED   			  = "locked";
+	public static final String UNLOCKED 			  = "unlocked";
+	public static final String LOCKED_OR_UNLOCKED     = "both";
+
+	private final static Set<String> REQUIRES_LOCK_OPTION_SET =
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AModifiersOrderCheck)